classes/config.php

```diff
@@ -36,19 +43,38 @@
 class config {
 
     /**
-     * A list of configured field and their descriptions.
+     * A list of configured web service field and their descriptions.
+     * @var array
+     */
+    private $webservicefields = [];
+
+    /**
+     * A field to find a user in Moodle by.
+     * @var string
+     */
+    private $usermatchfield = 'username';
+
+    /**
+     * Fields map config data.
+     * @var array
+     */
+    private $datamapping = [];
+
+    /**
+     * Default auth method.
      * @var array
      */
-    private $fields = [];
+    private $defaultauth = 'manual';
```

Comment:
Can't find how it's used. Will it be used later in further pull requests?

Reply:
there is get_default_auth method

Reply:
Which is not used at this stage, right?